src/main/java/org/folio/service/impl/RecordRepositoryImpl.java

```diff
@@ -87,14 +87,14 @@ public Future<List<User>> findUsersByFieldValues(RecordUpdate recordUpdate, Stri
   }
 
   @Override
-  public Future<Boolean> updateUser(User user, String tenantId) {
+  public Future<Void> updateUser(User user, String tenantId) {
     user.setUpdatedDate(new Date());
 
     Promise<RowSet<Row>> promise = Promise.promise();
 
     pgClient(tenantId).update(USERS_TABLE, user, user.getId(), promise);
 
-    return promise.future().map(updateResult -> updateResult.size() == 1);
+    return promise.future().map(r -> null);
```

Comment:
return pgClient(tenantId).update(USERS_TABLE, user, user.getId()).mapEmpty();

Reply:
Thanks. I've used `mapEmpty` before and forgot about it, I will change this.